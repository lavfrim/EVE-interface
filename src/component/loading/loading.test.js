import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loading from './index';

describe('Loading', () => {
    it('should render correctly', () => {
        const wrapper = mount(
            <Loading />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

