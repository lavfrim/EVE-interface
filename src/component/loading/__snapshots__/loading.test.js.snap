// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
<Loading>
  <span>
    Loading...
  </span>
</Loading>
`;
